kubernetes/helm/templates/configmap.yaml

```diff
@@ -0,0 +1,165 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nebula-graphd
+  namespace: {{ .Values.namespace }}
+data:
+  nebula-graphd.conf: |
+    ########## basics ##########
+    # Whether to run as a daemon process  
+    --daemonize=false
+    # The file to host the process id
+    --pid_file=pids/nebula-graphd.pid
+    ########## logging ##########
+    # The directory to host logging files, which must already exists
+    --log_dir=logs
+    # Log level, 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
+    --minloglevel=2
```

Comment:
Should let user specify the level of log

Reply:
good point